docs/source/getting_started.rst

```diff
@@ -0,0 +1,270 @@
+   [2021-07-07 16:44:42,513] [Thread-4] [remoulade.worker.WorkerThread] [INFO] Finished Actor load / a201d61d-5c24-40aa-9b1b-bd8226221804 after 0.30ms.
+
+Next steps
+------
+
+In this part, we will take a more advanced example to demonstrate how to use Results and Redis to gather results from a group and pipe them to a single actor.
+
+First, you will need to install the dependencies for using Remoulade with Redis::
+
+   $ pip install -U 'remoulade[redis]'
+
+You will also need to install Redis_ or use Docker_ to run a Redis image::
+
+   $ docker run -d --name redis -p 6379:6379 redis
+
+.. _Redis: https://redis.io/
+
+In this example, we want to get the weather for multiple cities, and then load them all into a file by using one single function instead of calling multiple times the ``load`` function. In order to achieve this, we have to write a new function : ``load_multiple``, which takes a list of text to load instead of a single text::
+
+   @remoulade.actor
+   def load_multiple(text_list):
+       file = open("weather", "a")
+       for text in text_list:
+           file.write(text)
+       file.close()
+
+Our get_weather file should now look like this :
+
+.. code-block:: python
+   :caption: get_weather.py
+   :emphasize-lines: 4, 5, 13, 28, 30, 32
+
+
+   import requests
+   import remoulade
+   from remoulade.brokers.rabbitmq import RabbitmqBroker
+   from remoulade.results.backends import RedisBackend
+   from remoulade.results import Results
+
+
+   @remoulade.actor
+   def extract(city):
+      ...
+
+
+   @remoulade.actor(store_results=True)
+   def transform(response):
+       ...
+
+
+   @remoulade.actor
+   def load(text):
+      ...
+
+
+   @remoulade.actor
+   def load_multiple(text_list):
+       ...
+
+
+   result_backend = RedisBackend()
+   rabbitmq_broker = RabbitmqBroker(url="/")
+   rabbitmq_broker.add_middleware(Results(backend=result_backend))
+   remoulade.set_broker(rabbitmq_broker)
+   remoulade.declare_actors([extract, transform, load, load_multiple])
+
+Note that in order to be able to gather the results from ``transform``, we have to specify "store_results=True" in the actor decorator.
+A RedisBackend was also instantiated and added to the broker, and ``load_multiple`` was declared to the broker.
+
+Our objective can now be achieved by running the following commands::
+
+   >>> from get_weather import extract, transform, load_multiple
+   >>> from remoulade import pipeline, group
+   >>> cities = ['Paris', 'Lyon', 'Marseille', 'Lille', 'Bastia']
+   >>> grp = group([extract.message(city) | transform.message() for city in cities])
+   >>> pipe = grp | load_multiple.message()
+   >>> pipe.run()
+
+The worker terminal now shows the pipeline has been successfully run and the correct data has been written into the weather file::
+
+   Paris:
+   +22 °C
+   Lyon:
+   21 °C
+   Marseille:
+   +25 °C
+   Lille:
+   21 °C
+   Bastia:
+   +31 °C
+
+
```

Comment:
Other topics for after:
- scheduler
- superbowl
No need to do that right now, but if you are bored it's an idea